inc/saiacl.h

```diff
@@ -265,6 +309,14 @@ typedef enum _sai_acl_table_attr_t
     SAI_ACL_TABLE_ATTR_STAGE = SAI_ACL_TABLE_ATTR_START,
 
     /**
+     * @brief List of ACL bind point where this ACL can be applied
+     *
+     * @type sai_acl_bind_point_t 
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
```

Comment:
@AnbalaganNatchimuthu - done in the updated pull request. 
@itaibaz - done in the updated pull request.